metrics/METRICS.md

```diff
@@ -0,0 +1,80 @@
+# Deploying on OpenShift
+
+## Prometheus
+
+The Prometheus server configuration uses a service discovery feature in order to discover the pods in the cluster from which getting metrics.
+In order to have this feature working, it's needed to provide access to the API server for getting the pods list to the service account used for running the Prometheus service pod (by default the used service account is `default`)
+
+```
+export NAMESPACE=[namespace]
+oc login -u system:admin
+oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${NAMESPACE}:default
+oc login -u developer
```

Comment:
Wouldn't it be better to create a special service account for Prometheus and give it the rights? Instead of giving the rights to all applications running in the nmespace?

Reply:
I left in this way because I understood that it should be just an example and nothing to use in production (as you said before and I agree on that) so nothing more than the minimum for having it working. Btw I agree with you that using a different service account could be better. I'll do that.

Reply:
Yes I have created the `prometheus-server` account and configured in the OS template.